packages/@aws-cdk/aws-appsync/test/appsync-auth.test.ts

```diff
@@ -0,0 +1,603 @@
+import '@aws-cdk/assert/jest';
+import * as path from 'path';
+import * as cognito from '@aws-cdk/aws-cognito';
+import * as cdk from '@aws-cdk/core';
+import * as appsync from '../lib';
+
+// GIVEN
+let stack: cdk.Stack;
+beforeEach(() => {
+  stack = new cdk.Stack();
+});
+
+describe('AppSync API Key Authorization', () => {
```

Comment:
I don't mind splitting out the `describes` into their own file. - I'll leave it up to you
it can get pretty large pretty fast

Reply:
i dont think it will get any larger to be honest.. i tried to make sure I covered the swathe of all possible unit tests